src/fund/utils/getCoinbaseSmartWalletFundUrl.ts

```diff
@@ -0,0 +1,21 @@
+import { version } from '../../version';
```

Comment:
This code was just extracted out of the original wallet dropdown fund link and moved into a util.